dashboard/pkg/epinio/dialog/ExportAppDialog.vue

```diff
@@ -94,19 +126,40 @@ export default {
         </Tab>
 
         <Tab
-          label-key="epinio.applications.export.chart.title"
+          label-key="epinio.applications.export.chartValuesImages.title"
           name="chart"
           :weight="2"
           class="export-app-dialog__tab"
         >
           <p>
-            {{ t('epinio.applications.export.chart.description') }}
+            {{ t('epinio.applications.export.chartValuesImages.description') }}
           </p>
           <Banner
             color="info"
           >
-            {{ t('epinio.applications.export.chart.banner') }}
+            {{ t('epinio.applications.export.chartValuesImages.banner') }}
           </Banner>
+
+          <div
+            v-if="showProgressBar"
+            class="zip-parts text info mb-10 mt-20"
+          >
+            <span
+              v-if="progressBar !== 100"
+            >
+              {{ t('epinio.applications.export.chartValuesImages.stats.downloadFiles') }}: {{ t(`epinio.applications.export.chartValuesImages.stats.parts.${ downloadingPart }`) }}
+            </span>
+            <PercentageBar
+              v-if="progressBar !== 100"
+              class="progress-bar"
+              :value="progressBar"
+              :color-stops="colorStops"
+              preferred-direction="MORE"
+            />
+            <span v-if="progressBar === 100">
+              {{ t('epinio.applications.export.chartValuesImages.stats.zipFiles') }}
```

Comment:
could this message be expanded.

`Compressing Files. This may take a long time...`

Reply:
Make sense, label updated.